lib.nix

```diff
@@ -12,10 +12,23 @@ let
        then result
        else default;
 
+  filterSource = src:
```

Comment:
There's a `builtin.` by the same name. Using a different name, like `filterSrc`, may reduce confusion/mistakes.

Reply:
OK, will rename to `localLib.cleanHaskellSource`.